<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPopUp</class>
 <widget class="QWidget" name="SettingsPopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SettingsPopUp</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(196, 236, 236);</string>
  </property>
  <widget class="QFrame" name="settings_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>340</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(
    x1: 0, y1: 0, x2: 1, y2: 1,
    stop: 0 rgba(39, 113, 90, 1), stop: 1 rgba(210, 254, 254, 1)
);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>290</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Arial&quot;;
color: white; 
background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(25, 81, 68, 1), stop: 1 rgba(57, 148, 127, 1)
);

border-radius: 10%;</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Arial&quot;;
color: white; 
background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(25, 81, 68, 1), stop: 1 rgba(57, 148, 127, 1)
);

border-radius: 10%;</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: 18pt &quot;Arial&quot;;
background:none;</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Arial&quot;;
color: white;

background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(0, 12, 19, 1), stop: 1 rgba(0, 82, 136, 1)
);

border-radius: 10%;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QTextEdit" name="sampleFreqText">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border-radius: 10px;
background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(0, 12, 19, 1), stop: 1 rgba(0, 82, 136, 1)
);
padding: 3px;
vertical-align: center;
text-align: center;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sampleFreqLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Arial&quot;;
color: white;

background: none;

border-radius: 10%;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Sample Frequency</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Arial&quot;;
color: white;

background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(0, 12, 19, 1), stop: 1 rgba(0, 82, 136, 1)
);

border-radius: 10%;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="sampleFreqText_2">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>-1</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border-radius: 10px;
background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(0, 12, 19, 1), stop: 1 rgba(0, 82, 136, 1)
);
padding: 3px;
vertical-align: center;
text-align: center;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="windowSizeLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Arial&quot;;
color: white;

background: none;

border-radius: 10%;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Window Size</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="windowSizeText">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border-radius: 10px;
background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(0, 12, 19, 1), stop: 1 rgba(0, 82, 136, 1)
);
padding: 3px;
vertical-align: center;
text-align: center;</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Arial&quot;;
color: white;

background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(0, 12, 19, 1), stop: 1 rgba(0, 82, 136, 1)
);

border-radius: 10%;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QTextEdit" name="sampleFreqText_3">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>-1</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border-radius: 10px;
background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(0, 12, 19, 1), stop: 1 rgba(0, 82, 136, 1)
);
padding: 3px;
vertical-align: center;
text-align: center;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSlider" name="minfreq_slider">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="minFreqLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Arial&quot;;
color: white;

background: none;

border-radius: 10%;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Min. Frequency</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Arial&quot;;
color: white;

background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(0, 12, 19, 1), stop: 1 rgba(0, 82, 136, 1)
);

border-radius: 10%;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="sampleFreqText_4">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>-1</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border-radius: 10px;
background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(0, 12, 19, 1), stop: 1 rgba(0, 82, 136, 1)
);
padding: 3px;
vertical-align: center;
text-align: center;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxFreqLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Arial&quot;;
color: white;

background: none;
   

border-radius: 10%;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Max. Frequency</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="maxfreq_slider">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>frame_3</zorder>
   <zorder>frame_2</zorder>
   <zorder>saveButton</zorder>
   <zorder>cancelButton</zorder>
   <zorder>titleLabel</zorder>
   <zorder>frame_4</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
